.github/workflows/integration-build-robotics-component.yml

```diff
@@ -0,0 +1,525 @@
+        shell: bash
+        run: |
+          ssh ${{ steps.infra_outputs.outputs.ssh_connect_string }} -i ~/.ssh/ci_priv_key.pem -o StrictHostKeyChecking=no '
+            set -e
+
+            export DISPLAY="$(w -oush | grep -Eo " :[0-9]+" | uniq | cut -d \  -f 2)"
+            echo "Display detected (dinamic) is $DISPLAY"
+            export DISPLAY=":0"
+            echo "Display detected is $DISPLAY"
+
+            cd ${{ env.REMOTE_WORKSPACE_PATH }}
+
+            export PATH="$HOME/.local/bin:$PATH"
+
+            PRODUCT_NAME="$(echo ${{ inputs.vs_product_name }} | sed '\''s/^project-\(movai-\)\?//'\'' )"
+            if [[ "$PRODUCT_NAME" == "studio" ]]; then
+              PRODUCT_NAME="solution"
+            fi
+            export CONFIG_FILE_NAME="product/standalone-spawner-$PRODUCT_NAME-simulator-noetic.json"
+
+            cp -r ${{ env.CI_INTEGRATION_EXECUTION_FOLDER }}/* ./ && sudo rm -rf ${{ env.CI_INTEGRATION_EXECUTION_FOLDER }}
+
+            export USERSPACE_FOLDER_PATH="${{ env.USERSPACE_FOLDER_PATH }}"
+            sudo mkdir -p $USERSPACE_FOLDER_PATH/.git;sudo chown 1000:1000 $USERSPACE_FOLDER_PATH/ -Rf
+            export PUBLIC_IP="$(hostname -I | cut -d " " -f1)"
+            export SIMULATION_ID="${{ env.SIMULATION_ID }}"
+            export ROBOTNAME="ci_${{ inputs.component_name }}"
+            sudo apt install jq -y
+            jq --version
+            jq '\''.robot.name = "$ROBOTNAME"'\'' $CONFIG_FILE_NAME > tmp.json && mv tmp.json $CONFIG_FILE_NAME
+
+            # Enable debug logs of movai-service
+            sudo mkdir -p /etc/movai/
+            sudo sh -c "echo MOVAI_SERVICE_LOGGER_LEVEL=DEBUG >> /etc/movai/config.env"
+            sudo chmod -R +r /etc/movai
+
+            ls -la ${{ env.USERSPACE_FOLDER_PATH }}
+
+            wget https://movai-scripts.s3.amazonaws.com/QuickStart_$(cat product/quickstart_version.txt).bash
+            chmod +x ./QuickStart_$(cat product/quickstart_version.txt).bash
+            ./QuickStart_$(cat product/quickstart_version.txt).bash --apps $(cat product/service_version.txt) $CONFIG_FILE_NAME
+            MOVAI_USER="ci"
+            MOVAI_PWD="4Iva6UHAQq9DGITj"
+            for robot in $(movai-cli robots list); do
+              movai-cli robots user "$robot" "$MOVAI_USER" "$MOVAI_PWD"
+            done
+
+            execution_status=$?
+            exit $execution_status
+            rm service_version.txt
+          '
+          echo "movai_user=ci" >> $GITHUB_OUTPUT
+          echo "movai_pwd=4Iva6UHAQq9DGITj" >> $GITHUB_OUTPUT
+          echo "robot_name=ci_${{ inputs.component_name }}" >> $GITHUB_OUTPUT
+
+
+      - name: Remote - Run tests
+        shell: bash
+        run: |
+          ssh ${{ steps.infra_outputs.outputs.ssh_connect_string }} -i ~/.ssh/ci_priv_key.pem -o StrictHostKeyChecking=no '
+            set -e
+
+            cd ${{ env.REMOTE_WORKSPACE_PATH }}/stack_tests
+
+            # setup tests venv in a step that is always executed
```

Comment:
Something is off with venv setup, we could only get it working by adding the setup here